activesupport/test/core_ext/numeric_ext_test.rb

```diff
@@ -310,6 +310,30 @@ def test_to_fs__human_size
     assert_equal "10 Bytes",  10.to_fs(:human_size)
   end
 
+  def test_to_fs__human_size_with_negative_number
+    assert_equal "-3 Bytes",   -3.14159265.to_fs(:human_size)
```

Comment:
I see that some other test cases has lots of assertions, but I think that for this (and other added test case) just a few edge case assertions would be enough.